EpiAware/test/test_autoregressive.jl

```diff
@@ -0,0 +1,62 @@
+@testitem "Testing AR constructor" begin
+    using Distributions
+
+    damp_prior = [truncated(Normal(0.0, 0.05), 0.0, 1)]
+    std_prior = truncated(Normal(0.0, 0.05), 0.0, Inf)
+    init_prior = Normal()
+    ar_process = EpiAware.AR(damp_prior, std_prior, init_prior)
+
+    @test ar_process.damp_prior == damp_prior
+    @test ar_process.std_prior == std_prior
+    @test ar_process.init_prior == init_prior
+end
+
+@testitem "Test AR defaults" begin
```

Comment:
Could you test this for creating a vector length `damp` variable?

Reply:
Good spot your right it isn't. I will follow the pattern in #134 and reping for a review.